ethapi/api.go

```diff
@@ -680,6 +681,30 @@ func (s *PublicBlockChainAPI) CurrentEpoch(ctx context.Context) hexutil.Uint64 {
 	return hexutil.Uint64(s.b.CurrentEpoch(ctx))
 }
 
+// GetRules returns network rules for an epoch
+func (s *PublicBlockChainAPI) GetRules(ctx context.Context, epoch rpc.BlockNumber) (interface{}, error) {
```

Comment:
why `(s *PublicBlockChainAPI) GetRules` returns `(interface{}, error)` instead of `(opera.Rules, error)`?